website/guides/workflow.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Workflow
+description: How does gql.tada and its LSP fit into your wofklow?
+---
+
+# Workflow
+
+There are some general workflows that are distinct locally (i.e. the LSP can take care of it)
+vs remotely. In this guide we'll look at how we can get local/remote to being consistent.
+
+> [!NOTE]
+>
+> This assumes that every `gql.tada` file is checked into VC.
+
+## Staying up to date
+
+There is a risk to go out of date when i.e. the schema is externally updated or when the LSP
+isn't running. Out of date could mean that the server has changed the schema, that the tada-output
+wasn't updated with the new schema or that the new persisted-operations weren't generated.
+
+To avoid running into this issue we encourage folks to leverage the `generate-schema`, `generate-persisted` and `generate-output`
+CLI commands. You can leverage these in CI and check whether a diff is present. 
+
+An example could look like the following
+
+```yaml
+- name: Verify schema and tada types are up to date
+  run: |
+    npx gql-tada generate-schema http://example.com
+    npx gql-tada generate-output -c ./tsconfig.json
+    npx gql-tada generate-persisted ./persisted-ops.json
+    git diff --name-only --exit-code -- .
+```
```

Comment:
`generate-persisted` is unrelated here, so I'd remove it? It's more something that's needed to be passed to other scripts/tasks

Reply:
There's no indication here that this is using GitHub workflow instance.

I'd maybe pull all workflow examples into a sub-section at the end of this page, and just keep the commands generic here.

Reply:
Re. `generate-schema`, also potentially not relevant here, since it only applies when the schema isn't easily accessible, so let's maybe keep that out of a CI-leaning section?

I think we can have a section above re. external/remote schemas (i.e. schemas on URLs that aren't easily accessible or are in separate repos with the recommendation to modify the config to a local SDL file and to use the CLI)